HookHander/AbstractHookHandler.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Copyright 2013 Zikula Foundation.
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license MIT
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\DizkusModule;
```

Comment:
this directory has a typo `Hander` vs `Handler` and also the namespace should include the parent directory e.g. `namespace Zikula\DizkusModule\HookHandler`